src/seismometer/table/analytics_table.py

```diff
@@ -0,0 +1,667 @@
+        per_context: bool = False,
+    ):
+        """
+        Initializes the AnalyticsTable object with the necessary data and parameters.
+
+        Parameters
+        ----------
+        score_columns: Optional[List[str]]
+            A list of column names corresponding to model prediction scores, by default None.
+        target_columns: Optional[List[str]]
+            A list of column names corresponding to (binary) targets, by default None.
+        metric: str
+            Performance metrics will be presented for the provided values of this metric.
+        metric_values: List[float]
+            Values for the specified metric to derive detailed performance statistics, by default [0.1, 0.2].
+        metrics_to_display: Optional[List[str]]
+            List of metrics to include in the table, by default None. The default behavior is to show all columns
+            in GENERATED_COLUMNS.
+        title: str
+            The title for the performance statistics table, by default "Model Performance Statistics".
+        top_level: str
+            The primary grouping category in the performance table, by default 'Score'.
+        decimals: int
+            The number of decimal places for rounding numerical results, by default 3.
+        table_config: Optional[AnalyticsTableConfig]
+            Configuration for the analytics table, including formatting and display options.
+        statistics_data: Optional[pd.DataFrame]
+            Additional performance metrics statistics, will be joined with the statistics data generated
+            by the code, by default None.
+        per_context : bool
+            If scores should be grouped by context, by default False.
+
+        Raises
+        ------
+        ValueError
+            If neither "sg.dataframe" nor "statistics_data" has data.
+            If "sg.dataframe" has data but either "score_columns" or "target_columns" is not provided.
+            If the provided metric name is not recognized.
+            If the provided top_level name is not "Score" or "Target".
+        """
+        sg = Seismogram()
+        self.df = sg.dataframe
+        self.score_columns = score_columns or sg.output_list
+        self.target_columns = target_columns
+        if sg.dataframe is not None and sg.target_cols:
+            self.target_columns = self.target_columns or sg.get_binary_targets()
+        self.statistics_data = statistics_data
+        if self.df is None and self.statistics_data is None:
+            raise ValueError("At least one of 'df' or 'statistics_data' needs to be provided.")
+        if self.df is not None:
+            if not self.score_columns or not self.target_columns:
+                raise ValueError(
+                    "When df is provided, both 'score_columns' and 'target_columns' need to be provided as well."
+                )
+
+        self.decimals = table_config.decimals
+        self.metric = metric
+        self.metric_values = metric_values
```

Comment:
```suggestion
        self.metric_values = list(set(metric_values))
```

This fixes the issue where the range slider matches itself

Reply:
FINC.